package.json

```diff
@@ -60,7 +60,7 @@
     "@rollup/plugin-node-resolve": "7.0.0",
     "@types/jest": "^26.0.14",
     "@types/json-schema": "^7.0.3",
-    "@types/node": "^14.11.2",
+    "@types/node": "10.14.0",
```

Comment:
looks like a downgrade, what happens here?

Reply:
looks like SIW need to explicitly install @types/node, https://github.com/okta/okta-developer-docs/pull/2284/files#diff-25fddbf1b579b42f71891a504022c8ea3c09ad6b9697ecc38e2e47de3fea43d6R63

can you explain more about that? maybe we want it be in dependencies?

Reply:
it's fine if we don't depend on it in bundles.